### PR TITLE
New privacy page

### PR DESCRIPTION
**Pull request by PierrePaul**
I'm not a big fan of putting so much HTML in a .po file, but at least it's consistent with our other translations for now.

### CODE REVIEW

**Comment by PierrePaul** [comment]
Missing link on the homepage.

File: profiles/templates/includes/footer.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
             <ul>
                 <li><a href="#">{% trans "Contact" %}</a></li>
                 <li><a href="#">{% trans "Terms and conditions" %}</a></li>
```
Comment (pcraig3) [inline profiles/templates/includes/footer.html:9]: Talked to Courtney and we decided to remove the links that don't go anywhere from the footer until those pages are ready.

File: profiles/templates/profiles/landing.html [hunk 2]
```diff
@@ -18,7 +18,8 @@ <h1>{% trans "Welcome to Generate COVID Alert number" %}</h1>
 
   <div class="content-button"><a href="{% url 'login' %}" role='button' draggable='false'>{% trans "Log in" %}</a></div>
 
-  <p>{% trans "About COVID Alert [hyperlink to site homepage]" %}</p>
-  <p>{% trans "How COVID Alert protects privacy [hyperlink to privacy policy]" %}</p>
-
+  {% url 'landing' as landing_url %}
+  <p>{% blocktrans %}<a href="{{ landing_url }}">About COVID Alert</a>{% endblocktrans %}</p>
+  {% url 'privacy' as privacy_url %}
+  <p>{% blocktrans %}<a href="{{ privacy_url }}">How COVID Alert Portal protects privacy</a>{% endblocktrans %}</p>
```
Comment (pcraig3) [inline profiles/templates/profiles/landing.html:24]: Interesting, I haven't seen it done this way before. Works fine though 👍

File: locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -379,13 +381,385 @@ msgstr ""
...
-#: profiles/templates/profiles/landing.html:21
-msgid "About COVID Alert [hyperlink to site homepage]"
-msgstr "À propos d’Alerte COVID [lien]"
-
 #: profiles/templates/profiles/landing.html:22
-msgid "How COVID Alert protects privacy [hyperlink to privacy policy]"
-msgstr "Comment Alerte COVID protège la vie privée"
+#, python-format
+msgid "<a href=\"%(landing_url)s\">About COVID Alert</a>"
+msgstr "<a href=\"%(landing_url)s\">À propos d'Alerte COVID</a>"
```
Comment (pcraig3) [inline locale/fr/LC_MESSAGES/django.po:387]: Let's use the curly apostrophe for this. ie, `’`, not `'`

**Comment by pcraig3** [comment]
Awesome!

**Pull request merged**